src/widgets/JSONForm/JSONForm.js

```diff
@@ -130,6 +130,8 @@ export const JSONForm = React.forwardRef(
     { formData, onChange, theme = defaultTheme, children, options, onSubmit, surveySchema },
     ref
   ) => {
+    if (!surveySchema) return null;
```

Comment:
complaints about `forwardRef` being passed an invalid value for `value`

Reply:
ahhhh